app/views/users/index.html.erb

```diff
@@ -1,10 +1,13 @@
-<% if @users.empty? %>
-  <h3>Please Add Users</h3>
-<% end %>
 <div class="container">
+  <h1>Users</h1>
+  <% unless @users.empty?%>
     <% @users.each do |user| %>
         <a href="/users/<%= user.id%>" class="text-decoration">
             <%= render 'shared/user', user: user %>
         </a>
-   <% end %>    
+   <% end %>
+   <%end%>
+       <p>Sign up or Sign in and see </p> 
+         <%= link_to 'Sign up', new_user_registration_path%>
+         <%= link_to 'Sign In', new_user_session_path%>
 </div>
```

Comment:
- [OPTIONAL] I noticed that after I signed up, I still see these buttons. You don't need them to be displayed when a user is signed in, instead, the user should see the sign-out button. You can use the `user_signed_in` devise helper to control what users see when they are not signed in or when they are signed in.

```ruby
<% if user_signed_in? %>
  <%= link_to 'Log Out', destroy_user_session_path, method: :delete %>
<% else %>
  <%= link_to 'Sign Up', new_user_registration_path %>
  <%= link_to 'Log In', new_user_session_path %>
<% end %>

```

Reply:
It Would be nice if you addressed this issue because optional suggestion are important and they help improve your coding skills